svm/tests/example-programs/write-to-account/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    entrypoint,
+    entrypoint::ProgramResult,
+    incinerator, msg,
+    program_error::ProgramError,
+    pubkey::Pubkey,
+};
+
+entrypoint!(process_instruction);
+
+fn process_instruction(
+    _program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    data: &[u8],
```

Comment:
Does `data` contains some random number to prevent duplicate txs? This is not the only way but the simplest

Reply:
no check on length - if need that functionality easy to just append random bytes at the end; program has no need to read them

Reply:
no, duplicates arent a concern because all the svm integration tests are structured like "here is a specific list of transactions to execute, heres what the execution results, final account states, and log messages should be"